### ISSUE TITLE
Screen Readers: Paid expense details: The button 'back' is announced non-descriptively

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#76959]
## Action Performed:
1. Using Chrome+JAWS or NVDA + Firefox, open the URL: https://new.expensify.com/
2. Navigate using the tab key to the Create expense button on the Left side navigation and press Enter to activate it.
3. Tab to focus on the Currency field and enter an amount(eg, 250) and activate the Next button.
4. In the Confirm dialog, Tab to focus on the Merchant button and press Enter to activate it.
5. Tab to focus on the Edit field and enter the Merchant name, and save it.
6. Tab to focus on the Create expense button and press Enter to activate it.
7. In the Inbox tab, focus on the created expense message and activate it.
8. Tab to focus on the More dropdown button and press Enter to activate it.
9. Tab to focus on the Export menu item and press Enter to activate it.
10. In the Export pop-up, navigate to the Back button and observe the screen reader announcement.

## Expected Result:
Each button should have a clear, descriptive label that conveys both purpose and direction, such as: “Back to previous screen”, “Export report”. The arrow icon’s purpose (if decorative) should be hidden from assistive technologies using aria-hidden="true", or if meaningful, it should be included in the accessible name.

## Actual Result:
When the "exports" button with icon > indicating "back to exports" receives focus, screen readers only announce the text content, "Export", and completely ignore the navigational context provided by the icon. This results in an incomplete and misleading announcement.

HTML:
<div aria-label="Export" role="menuitem" data-tag="pressable" tabindex="0" class="css-g5y9jx r-1otgn73 r-1loqt21 r-17gur6a r-18u37iz r-1wtj0ep r-1mmae3n r-u9wvl5 r-13qz1uu r-1mdbw0j r-wk8lta" data-component="GenericPressable" data-element="Pressable" data-source-file="BaseGenericPressable.tsx"><div class="css-g5y9jx r-13awgt0"><div class="css-g5y9jx r-18u37iz"><div class="css-g5y9jx r-eqz5dr r-13awgt0"><div class="css-g5y9jx r-18u37iz r-105ug2t"><div class="css-g5y9jx r-1awozwy r-1777fci" style="width: 40px;"><div class="css-g5y9jx r-icoktb r-1d4mawv" data-testid="" style="width: 16px; height: 16px;"><svg xmlns="[http://www.w3.org/2000/svg"](http://www.w3.org/2000/svg%22) xml:space="preserve" viewBox="0 0 20 20" width="20" height="20" hovered="false" pressed="false" fill="#8B9C8F"><path d="M14.2 18.4c-.8.8-2 .8-2.8 0L3 10l8.4-8.4c.8-.8 2-.8 2.8 0s.8 2 0 2.8L8.7 10l5.6 5.6c.7.8.7 2-.1 2.8" style="fill-rule: evenodd; clip-rule: evenodd;"></path></svg></div></div><div class="css-g5y9jx r-1777fci r-13awgt0" style="min-height: 40px;"><div class="css-g5y9jx r-18u37iz r-1awozwy r-dnmrzs"><div dir="auto" data-selection-scrapper-hidden-element="false" class="css-146c3p1 r-1udh08x r-1udbk01 r-1iln25a" style="color: rgb(175, 187, 176); font-size: 13px; text-align: left; font-family: &quot;Expensify Neue&quot;, &quot;Windows Segoe UI Emoji&quot;, &quot;Noto Color Emoji&quot;, &quot;Custom Emoji Font&quot;; font-style: normal; font-weight: 400; flex-shrink: 1; margin-left: 12px; white-space: pre; direction: ltr; max-width: 100%;">Export</div></div></div></div></div><div class="css-g5y9jx r-18u37iz" style="min-height: 40px;"></div></div></div></div>

## Area issue was found in:
Paid Expense Details

## Failed WCAG checkpoints
WCAG 2.4.6 Headings and Labels
https://www.w3.org/WAI/WCAG22/Understanding/headings-and-labels

## User impact:
Descriptive headings are necessary for users who have disabilities that make reading slow and for people with limited short-term memory. These users benefit when section titles make it possible to predict what each section contains. Form input controls with labels that clearly describe the content that is expected to be entered helps users know how to successfully complete the form.

## Suggested resolution:
Update the "aria-label" attribute of the button to something descriptive as "back to exports" or something similar.
https://www.w3.org/WAI/ARIA/apg/patterns/button/

## Environment(s)
Windows Windows 11 - Firefox

## Screenshots/Videos
https://github.com/user-attachments/assets/cf7995e2-0cf8-47c6-af36-732bdd5212e5

<img width="1919" height="926" alt="Image" src="https://github.com/user-attachments/assets/79f8fcea-6170-410a-8e13-61d6ebd969ab" />

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/7000220
**Issue reported by:** NA
**Slack conversation:** NA
